cuppa/src/main/python/pycuppa/cuppa/visualization/plot_predictions.R

```diff
@@ -67,7 +67,9 @@ get_plot_data_probs <- function(VIS_DATA, data_value_rounding = 2){
 
    plot_data <- subset(VIS_DATA, data_type=="prob")
 
-   plot_data$row_label <- toupper(plot_data$clf_name)
+   plot_data$row_label <- toupper(gsub("_"," ", plot_data$clf_name))
+   plot_data$row_label[plot_data$row_label == "GENE EXP"] <- "GENE EXPRESSION"
+   plot_data$row_label[plot_data$row_label == "GEN POS"] <- "GENOMIC POSITION"
```

Comment:
It would be better to define mappings that the top of the script, e.g.:

```
MAPPINGS_CLASSIFIER_NAMES = c(
  GENE_EXP="GENE EXPRESSION",
  GEN_POS="GENOMIC POSITION"
)
```

and then use a function like `revalue` from `plyr` (or write a custom function):
`plyr::revalue(plot_data$row_label, replace=MAPPINGS_CLASSIFIER_NAMES)`
See: https://www.rdocumentation.org/packages/plyr/versions/1.8.9/topics/revalue

I would also define `MAPPINGS_FEAT_NAMES`

Reply:
@luan-n-nguyen Thanks for the tip on the revalue function! 

Re `MAPPINGS_FEAT_NAMES`: Do you mean to explicitly map every feature name to its non underscored version instead of using `gsub("_", " ", ..)`?
```
MAPPING_FEAT_NAMES = c(
      snv_count="snv count"
      is_male="is male",
      whole_genome_duplication="whole genome duplication",
      ....
)
```

Reply:
Oh never mind this comment. For the feature names it's indeed better to use some substitutions / regex rules